docs/CHANGELOG.asciidoc

```diff
@@ -44,6 +45,265 @@ Review important information about the {kib} 8.x releases.
+* Setup ignore_malformed in fleet {kibana-pull}157184[#157184]
+
+Lens & Visualizations::
+* Adds several new capabilities for annotation groups in *Lens* {kibana-pull}152623[#152623]
+
+Observability::
+* Adds SLO create callout to service overview, transactions page and transactions details {kibana-pull}159958[#159958]
+* Adds the Logs threshold alert detail page, which provides more information and context about the Logs threshold alert {kibana-pull}159947[#159947]
+
+Security::
+* Adds vulnerability dashboard tables {kibana-pull}159699[#159699]
+* Adds new Vulnerabilities tab to the Group by Resource page {kibana-pull}158987[#158987]
+* Adds display errors and check licenses for actions in response actions {kibana-pull}155254[#155254]
+* Adds common response actions tab in the alert flyout {kibana-pull}155362[#155362]
+
+For more information about the features introduced in 8.9.0, refer to <<whats-new,What's new in 8.9>>.
+
+[[enhancements-and-bug-fixes-v8.9.0]]
+=== Enhancements and bug fixes
+
+For detailed information about the 8.9.0 release, review the enhancements and bug fixes.
+
+[float]
+[[enhancement-v8.9.0]]
+=== Enhancements
+Alerting::
+* Adds a Mustache lambda for alerting actions to format numbers with `{{#FormatNumber}}` using `Intl.NumberFormat` {kibana-pull}159644[#159644]
+* Removes bulk snoozing of rules in Select All mode {kibana-pull}159749[#159749]
+* Adds refresh button to maintenance windows list {kibana-pull}159618[#159618]
+* Adds the feature for a Slack API to have allowed list on channels {kibana-pull}159534[#159534]
+* Integrate Conditional Actions with several Observability rule types {kibana-pull}159522[#159522]
+* Adds AAD Fields API {kibana-pull}158516[#158516]
+* Adds API to retrieve the `fieldsForAAD` from a given rule type {kibana-pull}158516[#158516]
+* Improves the performance of clearing expired snooze schedules {kibana-pull}157909[#157909]
+APM::
+* Ensure Saved Objects are versionable {kibana-pull}159881[#159881]
+* Adds active alerts column for transaction group table {kibana-pull}159552[#159552]
+* Adds an OpenAI integration {kibana-pull}158678[#158678]
+* Adds Storage explorer improvements {kibana-pull}157303[#157303]
+* Adds logic to replace transaction histogram with summary for backwards compatibility {kibana-pull}155714[#155714]
+Dashboard::
+* Adds design enhancements to the clone experience in Dashboards {kibana-pull}159752[#159752]
+* Adds enhancements to the empty state screen in Dashboards {kibana-pull}158496[#158496]
+* Adds a query DSL documentation link to filters UI {kibana-pull}156543[#156543]
+* Adds a counter displaying the min/max values of the time series counter field in the field popover {kibana-pull}155499[#155499]
+* Adds ability to recover from non-fatal error state {kibana-pull}158087[#158087]
+Discover::
+* Updates Discover sharing capabilities to enable sharing a link to Discover when using ad-hoc (temporary) data views {kibana-pull}154947[#154947]
```

Comment:
```suggestion
* Updates Discover sharing capabilities to enable sharing a link when using temporary data views {kibana-pull}154947[#154947]
```
I'd suggest to go with the name in our docs, also it might be redundant to mention Discover a second time